fineract-provider/config/checkstyle/spotbugs-exclude.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements. See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership. The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied. See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<FindBugsFilter>
+    <Match>
+        <Bug pattern="ES_COMPARING_PARAMETER_STRING_WITH_EQ" />
```

Comment:
Unfortunately EclipseLink weaving makes some changes to classes which violate this particular spotbugs rule. Given that I can't fix this without disabling weaving/enhancement, I thought it best to just ignore.

Reply:
Perhaps instead of globally disabled there is a way to only have it exclude for `**.domain.**` classes? @percyashu @thesmallstar

Reply:
Ping @xurror , will you like to address this feedback?